src/stylesheets/components/_contact-panel.scss

```diff
@@ -0,0 +1,24 @@
+.app-c-contact-panel {
+  @include govuk-responsive-margin($govuk-spacing-responsive-6, "top");
+  @include govuk-responsive-padding($govuk-spacing-responsive-3);
+  background-color: $govuk-blue;
+}
+
+.app-c-contact-panel--heading {
```

Comment:
These are _elements_ not _modifiers_ so they should be using the double underscore (`__`) rather than double hyphens (`--`) as a separator.

https://github.com/alphagov/govuk-frontend/blob/master/docs/coding-standards/css.md#bem